test/fixture/fixtures/test.js

```diff
@@ -0,0 +1,6 @@
+import test from '../../';
```

Comment:
Great point—agreed. I'll do this after my flight. @sindresorhus after I make that change should I squash that commit with the rest?